<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodaySummaryWidget</class>
 <widget class="QWidget" name="TodaySummaryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2{
background-color: rgb(46, 52, 54)
}
QLabel
{
color:rgb(255,255,255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QProgressBar" name="prg_efficiency">
          <property name="font">
           <font>
            <family>Lato Heavy</family>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"> QProgressBar {
      border: 1px solid rgb(114, 159, 207);
border-radius:2px;
text-align: center;
  }

  QProgressBar::chunk {
      background-color: rgb(252, 175, 62);
      width: 20px;
  }</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>B Yekan</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(114, 159, 207);</string>
          </property>
          <property name="text">
           <string>درصد بهره‌وری امروز</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>16</horstretch>
            <verstretch>16</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/ui/res/statistics.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_change_day_start">
          <property name="toolTip">
           <string>تغییر شروع امروز</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/ui/res/edit.png</normaloff>:/ui/res/edit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_today_start">
          <property name="font">
           <font>
            <family>Lato Thin</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>34:56</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>B Yekan</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(52, 101, 164);</string>
          </property>
          <property name="text">
           <string>شروع امروز</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>16</horstretch>
            <verstretch>16</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/ui/res/statistics.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lbl_today_duration">
          <property name="font">
           <font>
            <family>Lato Thin</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>B Yekan</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(52, 101, 164);</string>
          </property>
          <property name="text">
           <string>کل کارکرد امروز</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>16</horstretch>
            <verstretch>16</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/ui/res/statistics.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lbl_total_useful_time">
          <property name="font">
           <font>
            <family>Lato Thin</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>B Yekan</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(52, 101, 164);</string>
          </property>
          <property name="text">
           <string>مدت زمان کار مفید</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>16</horstretch>
            <verstretch>16</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/ui/res/statistics.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
